src/geoapis/__init__.py

```diff
@@ -4,4 +4,4 @@
 
 @author: pearsonra
 """
-__version__ = "0.3.0"
+__version__ = "0.4.0"
```

Comment:
Do you think we could make this 0.3.1 as I don't think this constitutes a major functionality change :)